internal/runnerinstall/ecs.go

```diff
@@ -359,89 +356,110 @@ func (i *ECSRunnerInstaller) Uninstall(ctx context.Context, opts *InstallOpts) e
 			return err
 		}
 
-		s.Update("Runner uninstalled")
+		s.Update("Waypoint runner AWS ECS service deleted")
 	}
 	s.Done()
 
 	// TODO: Still attempt to delete the EFS volume if the ECS service
 	// uninstall fails
+	s = sg.Add("Deleting runner file system")
 	efsSvc := efs.New(sess)
 	marker := ""
 	done := false
 	var fileSystems []*efs.FileSystemDescription
 	for !done {
-		fileSystemsResp, err := efsSvc.DescribeFileSystems(&efs.DescribeFileSystemsInput{
-			Marker:   aws.String(marker),
+		req := &efs.DescribeFileSystemsInput{
 			MaxItems: aws.Int64(100),
-		})
+		}
+		if marker != "" {
+			req.Marker = aws.String(marker)
+		}
+		fileSystemsResp, err := efsSvc.DescribeFileSystems(req)
 		if err != nil {
 			return err
 		}
-		if marker == *fileSystemsResp.Marker {
+
+		if fileSystemsResp.NextMarker == nil {
 			done = true
+		} else {
+			marker = *fileSystemsResp.NextMarker
+			time.Sleep(5 * time.Second)
```

Comment:
We probably don't need the sleep here since we are not waiting for an installation or otherwise status change

Reply:
I added this sleep because during testing, I triggered the API's rate limiting. That was before I had this loop working correctly, though so it _shouldn't_ still happen. I will go ahead and remove this.